src/view/root_controller.go

```diff
@@ -0,0 +1,81 @@
+package view
+
+import (
+	"fmt"
+	"unicode"
+
+	"github.com/gdamore/tcell/v2"
+	"github.com/rivo/tview"
+	"github.com/wombatlord/last-player-on-the-left/src/app"
+	"github.com/wombatlord/last-player-on-the-left/src/lastplayer"
+)
+
+type RootContoller struct {
+	FlexController
+	logger chan string
+	view   *tview.Flex
+	gui    *tview.Application
+	focusRing []tview.Primitive
+}
+
+func NewRootController(gui *tview.Application) *RootContoller {
+	return &RootContoller{
+		gui:    gui,
+		logger: app.GetLogChan("RootController"),
+	}
+}
+
+func (r *RootContoller) SetFocusRing(focusRing []tview.Primitive) {
+	r.focusRing = focusRing
+}
+
+func (r *RootContoller) Attach(root *tview.Flex) {
+	r.view = root
+	root.SetInputCapture(r.InputHandler)
+}
+
+func (r *RootContoller) View() *tview.Flex {
+	return r.view
+}
+
+func (r *RootContoller) Receive(s app.State) {}
+
+func (r *RootContoller) InputHandler(event *tcell.EventKey) *tcell.EventKey {
+
+	
+	r.logger <- fmt.Sprintf("%+v", r.gui.GetFocus())

```

Comment:
Consider trimming out these new lines

```suggestion
func (r *RootContoller) InputHandler(event *tcell.EventKey) *tcell.EventKey {
	r.logger <- fmt.Sprintf("%+v", r.gui.GetFocus())

```